reference/carts.v3.yml

```diff
@@ -895,22 +881,43 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/MetafieldResponse'
-        '409':
-          description: |
-            The `Metafield` conflicts with another `Metafield`. This can result from duplicate unique key combinations of the appʼs client id, namespace, key, resource_type, and resource_id.
+        '400':
```

Comment:
I don't think we can get 400 on singular get. Just 200 or 404

Reply:
Yeah, let's remove all 400's in this PR for get endpoints :)